index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css">
+</head>
+<body>
+    <header class = "top">
+        <div class="head">
+           <img src="Logo.svg" class="logo">
+            <img src="Union.svg" class="union">
+        </div>
+<div class="paragraph">
+    <h1 class="intro">I'm Shoaib
+        Glad to see you!</h1>
+        <p class="detail">I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p></div>
+        <img src="Connect.svg" class="Connect">
+        <div class="links">
+          <img src="Twitter.svg" class="socialmedia"/>
+          <img src="Linkedin.svg"class="socialmedia"/>
+          <img src="Vector-1.svg"class="socialmedia"/>
+          <img src="github.svg"class="socialmedia"/>
+          <img src="Vector.svg"class="socialmedia"/>
+        </div>
```

Comment:
- [ ] KIndly make sure to wrap your social media icons into the `ul` tag because of the semantic reason. :wink: 
- [ ] Kindly make sure to wrap each of the icons into a `li` tag. :wink: 
- [ ] Kindly make sure to warp each icon image in an `a` tag in order to increase the user experience. :wink:

Reply:
@levy002 resolved